contracts/SortitionPool.sol

```diff
@@ -178,12 +178,12 @@ contract SortitionPool is SortitionTree, Ownable {
       selected.arrayPush(leaf);
     }
 
-    address[] memory selectedAddresses = new address[](groupSize);
+    uint32[] memory selectedIDs = new uint32[](groupSize);
 
     for (uint256 i = 0; i < groupSize; i++) {
-      selectedAddresses[i] = selected.array[i].operator();
```

Comment:
ID is stored in the leaf:
https://github.com/keep-network/sortition-pools/blob/0e6a1b32fef28ce0f87fe1aa7ec7a1eb0401cd24/contracts/Leaf.sol#L58

Reply:
See https://github.com/keep-network/sortition-pools/pull/133/commits/b746f0f80b47daeb33c7728713b2382fcab3522e